docs/content/Getting-Started/user-quickstart-kind.md

```diff
@@ -0,0 +1,276 @@
+---
+short_name: user-quickstart-kind
+manifest_name: 'docs/content/Getting-Started/user-quickstart-kind.md'
+---
+[![User QuickStart Kind test]({{config.repo_url}}/actions/workflows/docs-ecutable-user-quickstart-kind.yml/badge.svg?branch={{config.ks_branch}})]({{config.repo_url}}/actions/workflows/docs-ecutable-user-quickstart-kind.yml)&nbsp;&nbsp;&nbsp;
+
+<!-- 
+!!! tip "Estimated time to complete this example:" 
+    ~4 minutes (after installing prerequisites) -->
```

Comment:
4 minutes isn't realistic unless someone is only going copy/paste without errors, and not reading any of the text along the way. 20 - 30 minutes is closer to a good pace.